### PR TITLE
Add calculator design

### PR DESCRIPTION
**Pull request by Bigizi**
**Math-Magician**

I created a component that contains the jsx for a calculator.
and imported it in App.js and styled it in App.css file.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,39 +1,58 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700;900&display=swap');
```
Comment (GzimAsani) [inline src/App.css:1]: - [ ] All the boilerplate that got created by the CRA (texts, images, and styles) should be deleted. Check the public folder

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      <button type="button">3</button>
+      <button type="button" className="highlight">+</button>
+      <button type="button" id="zero">0</button>
+      <button type="button">.</button>
+      <button type="button" className="highlight">=</button>
+    </div>
+  </div>
+);
+
+export default Calculator;
```
Comment (GzimAsani) [inline src/components/Calculator.js]: - [ ] According to the requirements the calculator component should have at least 1 child
image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello, @Bigizi              :wave:   
 Good job so far,  done There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with es6 :+1: 

 ## Required Changes 

- [ ] Your live link on the readme file is sending to the 404 page. If you have not created an online page kindly remove it and add a `To be added soon` sentence.
- [ ] Check the comments below :arrow_double_down: 

 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Nemwel-Boniface — APPROVED** [review]
## Hello @Bigizi         :wave: ,
I am Nemwel your reviewer for this PR :man_beard:  

Thank you for meeting all the project requirements good job! :partying_face: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :partying_face: 🎉 



## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Met all the milestone requirements ✔️ 

## [Optional] suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**